README.md

```diff
@@ -1,480 +1,133 @@
 
-```
-// Importing a single item using a relative path
-import ./lighting/pbr;
+**Test Cases** - A collection of examples to help tools interoperate.
 
-// Importing multiple items
-import my/geom/sphere/{ draw, default_radius as foobar };
+**WGSL Language Servers** - Enable syntax directed editing in IDEs.
+Shader development with IDEs should be well supported.
 
-// Imports a whole module. Use it with `bevy_ui.name`
-import bevy_ui/*;
-```
+**Syntax Highlighters** - Display pretty extended WGSL on web pages with
+tools like Shiki and CodeMirror.
 
-These can then be used anywhere in the source code.
+**More Tools** - We'd like to see documentation extractors (akin to rustdoc/javadoc),
+prettifiers, etc. that understand the extended version of WGSL.
 
-```
-fn main() {
-    bevy_ui.quad(vec2f(0.0, 1.0), 1.0);
-    let a = draw(3);
-}
-```
+## Stability
 
-Both `bevy_ui` and `my` are packages in the current project. Language servers and related tools can look in a `wgsl.toml` file to find the location of the packages. This lets libraries be published to package managers, and users can import them with a simple syntax.
+As a community project,
+we can iterate quickly on experimental WGSL features for projects to use.
 
-The first part is the file path, which is assumed to have a `.wgsl` file extension.
-TODO: Maybe we should use a different file extension, like `.wesl` for "WGSL Extended Shading Language".
+Stabilized features will be grouped in periodic releases
+with backwards compatibilty with the aim of
+nurturing an ecosystem of wgsl libraries.
 
-Recursive import definitions are also supported, which leads to short and succinct import statements.
+## Filename extension
 
-```
-import bevy_pbr/{
-  forward_io/VertexOutput,
-  pbr_types/{PbrInput, pbr_input_new},
-  pbr_bindings/*
-};
+Source files containing
+the community variant of wgsl-enhanced files
+should have the `.wesl` extension
+to distinguish them from vanilla W3C WGSL.
 
-fn main() {
+## WGSL Enhancements Roadmap
 
-}
-```
+Several other features are queued up for discussion.
+`import` is the first priority, the others are not prioritized.
 
-## `wgsl.toml` file
+Many projects already need these features:
 
-The `wgsl.toml` file is a configuration file that tells the language server where to find the packages. It is similar to a `Cargo.toml` file in that regard.
+* **[imports](./Imports.md)** -
+`import` enhanced WGSL fragments from other files and eventually from libraries.
 
-```toml
-name = "my"
-edition = "2024"
+* **[conditional compilation](./ConditionalCompilation.md)** -
+Select blocks of WGSL to enable based on external variables.
+e.g. `#if MOBILE_CLASS_GPU`
+or `#[cfg gpu_class="mobile")]`.
 
-[dependencies]
-bevy_ui = { cargo = "bevy_ui" }
-shader_wiz = { path = "./node_modules/shader_wiz/src/main.wgsl" }
-```
+* **[simple templating](./SimpleTemplating.md)** -
+Insert runtime
+variables into WGSL from Rust/JavaScript host code into WGSL. e.g. `#{WORKGROUP_SIZE}`,
```

Comment:
I think these should be set per package and able to be defined in the manifest file. Otherwise you're leaking namespaces in your dependency tree that might result in unintended side effects

Reply:
agreed. added comment on packaging page.